### DISCUSSION TITLE
Ohme Integration

### DISCUSSION THREAD

**Discussion by slimjosh76** [springfall2008/batpred#883] [Q&A]
Hi,

thee is now a HA integration for the Ohme EV charger that I have been using for a few weeks now and it works well… so far. 

There is a sensor: sensor.ohme_charge_slots that when the car is plugged in returns the planned charge slots in a very similar way to octopus planned dispatches. Is there a way to use this sensor in predbat? 

Thanks

John

**Comment by springfall2008** [comment]
> Hi,
> 
> 
> 
> thee is now a HA integration for the Ohme EV charger that I have been using for a few weeks now and it works well… so far. 
> 
> 
> 
> There is a sensor: sensor.ohme_charge_slots that when the car is plugged in returns the planned charge slots in a very similar way to octopus planned dispatches. Is there a way to use this sensor in predbat? 
> 
> 
> 
> Thanks
> 
> 
> 
> John
> 
> 

Yes correct, the Ohme configuration is documented in the Predbat manual

**Reply by springfall2008** [reply]
https://springfall2008.github.io/batpred/devices/

**Reply by slimjosh76** [reply]
Many thanks
